.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+npx lint-staged
+ng test --no-watch --no-progress --browsers=ChromeHeadless
```

Comment:
Use `npm` always, not `ng`. Angular cli isn't installed on every machine in the world

Reply:
we'll fix it when we transfer it to jest)